src/user_crate/mod.rs

```diff
@@ -194,11 +196,22 @@ fn parse_source_and_deps(code_and_deps: &str) -> eyre::Result<(syn::Block, toml:
         }
     }
 
+    code_block.push_str(" }");
+    #[cfg(feature = "verify")]
```

Comment:
Fundamentally I question the premise that what Postgres calls a "trusted" language would not be able to ever use what Rust calls `unsafe`. However, if used to execute _untrusted user code_, obviously it's not acceptable. In fact, a fair number of things that Postgres would probably allow a trusted user language to do are things you probably would not want to let untrusted user code do.

Thus the name "verify", as in "trust but verify".

I think there should be a few sliders, in essence, on PL/Rust's safety configurations, so you can set those at build time, install the resulting extension, and declare your extension to be `plrust` or `plrustu`, depending on your safety assessment for your database installation. We probably want to initially make things work as much as possible with a maximally secured version by default and probably make it so that almost all security features default to "on".

Reply:
I wonder if offering "capabilities" which the admin could tweak (ala https://github.com/bytecodealliance/cap-std) might be interesting...